app/middlewares/debuggerAPI.js

```diff
@@ -32,6 +32,18 @@ let socket;
 
 const workerOnMessage = message => {
   const { data } = message;
+
+  if (data && data.source === 'apollo-devtools-backend') {
```

Comment:
If we get at least one message from apollo backend (the code in the worker) it means that apollo client is being used in the app. So we tell apollo dev tools to display its panel in chrome dev tools.

Also we forward the message from webworker to react native debugger web part.